content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -62,6 +62,10 @@ const shouldHide = process.env.PRISMA_HIDE_UPDATE_MESSAGE
 
 `PRISMA_GENERATE_SKIP_AUTOINSTALL` can be set to a truthy value to skip the auto-install of `prisma` CLI and `@prisma/client` dependencies (if they are missing), if the `prisma-client-js` generator is defined in the Prisma Schema, when using the `prisma generate` command.
 
+### <inlinecode>PRISMA_DISABLE_WARNINGS</inlinecode>
+
+Disables all CLI warnings.
```

Comment:
What warnings would that be? (I honestly do not know)

Reply:
They are all `logger.warn` https://sourcegraph.com/search?q=context:global+logger.warn%28+repo:%5Egithub%5C.com/prisma/prisma%24+&patternType=literal
It is useful for Redwood for example to avoid polluting their CLI